lib/reverse_sentence.rb

```diff
@@ -1,6 +1,40 @@
-# A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)
+# Space complexity: O(1)
+
 def reverse_sentence(my_sentence)
-  raise NotImplementedError
+  return nil if my_sentence == nil
+  length = my_sentence.length
+
+  reverse(my_sentence, 0, length - 1)
```

Comment:
Nice use of helper methods! 🙂

Nit: `reverse` could have a more descriptive name though - it's not _immediately_ clear (from the name alone) how it differs from `reverse_words` - perhaps something like `reverse_string` would be better?